apps/db/lib/friend_request/friend_request.ex

```diff
@@ -29,6 +29,11 @@ defmodule DB.FriendRequest do
     |> Repo.insert
   end
 
+  def create(requesting_user_id, requested_user_id)
```

Comment:
it feels more natural to me for this to be at the api layer

Reply:
That's fine, just as long as we don't introduce other codepaths for creating users. DB is our source of truth so I ended up putting it there for safety

Reply:
Oh I see you're point. A friend request should never be between the same user. Okay yea this makes a ton of sense

Reply:
Not sure but does the guard clause need to be above the other create function?